examples/client/banPeer.ts

```diff
@@ -32,7 +32,11 @@ const cli = meow(
     }
 );
 
-const [address, port] = cli.flags.endpoint.split(':');
+// Split endpoint on last colon
+const lastColonIndex = cli.flags.endpoint.lastIndexOf(':');
+const address = cli.flags.endpoint.substring(0, lastColonIndex);
+const port = cli.flags.endpoint.substring(lastColonIndex + 1);
+
```

Comment:
This is pretty much in every file. We should have a helper function to parse this instead, so we could have something like

```ts
const { address, port } = parseEndpoint(cli.flags.endpoint);
```

Note that that changes how the example should be run to `yarn run-example <path-to-script> [options]`, as otherwise loading local modules does not work, so we should update the description of how to run the script to match this.

I know this is annoying to have to change everywhere... Sorry! 🙈

Reply:
Shouldn't take too long

> _`sed` giveth, `sed` taketh away_